sites/fast-website/src/app/components/feature-card/feature-card.styles.ts

```diff
@@ -0,0 +1,69 @@
+import { css } from "@microsoft/fast-element";
+import { display, neutralForegroundRestBehavior } from "@microsoft/fast-components";
+
+export const FeatureCardStyles = css`
+    ${display("grid")} :host {
+        contain: layout;
+        grid-template-columns: 1fr 2fr;
+        color: var(--neutral-foreground-rest);
+        font-family: var(--text-font);
+        box-sizing: border-box;
+        padding: 20px;
+        box-shadow: unset;
+        height: 140px;
```

Comment:
If the content ever overflows the card here we may see it cut off. Let's consider how this works in responsive scenarios where the content might vary and increase the size.

Reply:
Understandable.  I was going off of the design initially here.  Kham said he was checking on what responsive behaviors design was looking for.  I will remove the height.  It's easy enough to put back in if need be.